src/pl/senla/hotel/entity/FreeRoomComparatorByPrice.java

```diff
@@ -0,0 +1,10 @@
+package pl.senla.hotel.entity;
+
+import java.util.Comparator;
+
+public class FreeRoomComparatorByPrice implements Comparator<FreeRoom> {
```

Comment:
Let's move comparators to separate package - .utils or .comparators or something like that. Because technically  they are not entyties

Reply:
ready